utils/dev-server.js

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Didn't realise that was a thing.

Main thing is to be able to start and use the server from unit tests. Made gulp serve over to using the test server and made test server use the serveStatic and serveIndex modules.

Only other change is to serve from the root of sw-helpers, this is to give the tests access to node_modules (i.e. sinon, mocha etc)